frontend/src/metabase/public/lib/analytics.ts

```diff
@@ -17,6 +17,8 @@ type Appearance = {
   theme: DisplayTheme;
   font: "instance" | "custom";
   hide_download_button: boolean | null;
+  // TODO:
```

Comment:
can you add more info why this to do is needed? e.g. it should be added, but it messes up `type-check`, so this will be fixed later, etc.

Reply:
22c5b6e6ca7ded58abaecc7856acaf28d14b5be1